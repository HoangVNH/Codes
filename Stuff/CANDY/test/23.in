10
22
XXXXXXVVXDDDDDDDDDXXXX
42
DDDDXXXXXDDDDVVXXXXXVVVVVDDDXXXXXXXDDXXXXX
28
DVVVVDDDDDDDDVVVVVVVVXXXXXXX
43
XXXDDDDVVXDDDDDDVVVVVXXXXXVVVVVVXVXXXXXDDDV
44
XXXXXDDDXXXXXDDVVVVDDDDVDDDDXXXXXXXXXXDDDVVV
30
DDDDDDDDXXVVVVXXXXVVVXXXXXVVXX
26
XXXXDDDDXXXVDDDDDDDDVVVVVV
45
XXDDDDXXXXDDDDDDDDDXXXXDXXXXDDDVDDDDVVVVVVXXX
39
XXXDVVXXXXXXXXXXXVVVVVVVVXXXXXXXXVDDDDD
22
DDDDDDDDDDXXXXXXVVVVVV
