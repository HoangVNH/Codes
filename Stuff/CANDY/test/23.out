21
41
27
41
42
29
25
43
38
20
